generation/new_client/README.md

```diff
@@ -312,6 +322,34 @@ generated Google Maps Routes API Java client library.
   --requires-billing=true \
   --distribution-name="com.google.maps:google-maps-routing"
 ```
+### Example with duplicate api_short_name
+
+Let's say you get a new library request where the Cloud Drop value for `api_short_name` is `maps-routing`.
+
+You discover that `maps-routing` is already in use by an existing client library!
+
+You need to determine a unique `destination_name` for the new library's subdirectory as well as a unique `distribution_name` for the artifact to be published (confirm these values with the service team on the buganizer ticket before proceeding).
+
+You will **still use** the non-unique `api_short_name` for the `api_short_name` flag.
```

Comment:
In this example, why is it important to use maps-routing-gps in api_shortname?
(An alternative is maps-routing-gps as api_shortname.)